webapp/webapp/demo/templates/train_annotations.html

```diff
@@ -28,6 +28,8 @@ <h5> Disclaimer </h5>
             <p>
             This software is intended solely for the testing purposes and non-commercial use. THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED.
             </p>
+              </p><a href="mailto:contact@cogstack.com">contact@cogstack.com</a> for more information.<p>
+
```

Comment:
The online form is referring to `contact@cogstack.org`: https://github.com/CogStack/MedCAT/blob/76cf2a52bb9f48c6c9ba2fd1a736d5d5f145db0b/webapp/webapp/demo/templates/umls_user_validation.html#L33
Can they be unified?